README.md

```diff
@@ -21,17 +21,17 @@
 ## Source Layout
 - `quic/api`:         Defines API that applications can use to interact with the QUIC transport  layer.
```

Comment:
Do use a favor and remove the extra space between "transport" and "layer"?

Reply:
Done!  Also did the same on the `congestion_control` line.